packages/engine/src/scene/systems/LinkSystem.ts

```diff
@@ -23,6 +23,7 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
+import { RouterService } from '@etherealengine/client-core/src/common/services/RouterService'
```

Comment:
the engine package cannot import client-core, the engine runs isomorphic across the client and server. if this requires manipulating client state, we should move the system to client-core (see how AvatarUISystem is started)